### PR TITLE
Add headless version and dockerfile

### PR DESCRIPTION
**Pull request by lsheva**
Closes https://github.com/opengs/uashield/issues/6

### CODE REVIEW

**Comment by opengs** [comment]
Will check this in a minute. Thanks

File: src-worker/doser.ts [hunk 1]
```diff
@@ -31,16 +31,28 @@ export class Doser {
...
 
   private workerActive: Array<boolean>
 
-  constructor (onlyProxy: boolean, workers: number) {
+  constructor (onlyProxy: boolean, workers: number, verboseError = true) {
     this.onlyProxy = onlyProxy
     this.working = false
     this.workers = workers
     this.eventSource = new EventEmitter()
     this.workerActive = new Array<boolean>(256)
```
Comment (opengs) [inline src-worker/doser.ts:43]: There must  be array with dynamic workers count, not 256

File: src-worker/doser.ts [hunk 2]
```diff
@@ -105,7 +115,7 @@ export class Doser {
     this.setWorkersCount(this.workers)
     for (let i = 0; i < 256; i++) {
```
Comment (opengs) [inline src-worker/doser.ts:116]: change 256 depending on max workers count

File: src-worker/doser.ts [hunk 3]
```diff
@@ -31,16 +31,28 @@ export class Doser {
...
+    this.verboseError = verboseError;
+  }
+
+  private logError(message:string, cause: unknown){
+    console.log(message);
+
+    if (this.verboseError){
+      console.log(cause)
+    } else{
+      console.log((cause as AxiosError)?.message)
```
Comment (yastasiuk) [inline src-worker/doser.ts:54]: You can guard TS with `isAxiosError`
example: https://github.com/axios/axios#typescript

**Pull request merged**